pkg/cmd/server/bootstrappolicy/policy_test.go

```diff
@@ -0,0 +1,72 @@
+package bootstrappolicy
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/ghodss/yaml"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/util"
+
+	"github.com/openshift/origin/pkg/api/v1"
+)
+
+func TestOpenshiftRoles(t *testing.T) {
+	roles := GetBootstrapOpenshiftRoles("openshift")
+	list := &api.List{}
+	for i := range roles {
+		list.Items = append(list.Items, &roles[i])
+	}
+	testObjects(t, list, "bootstrap_openshift_roles.yaml")
```

Comment:
Explain how you created these files in the error message so people know how to do it.